core/src/test/java/org/apache/iceberg/TestFastAppend.java

```diff
@@ -313,6 +313,37 @@ public void testRecoveryWithoutManifestList() {
         metadata.currentSnapshot().allManifests(FILE_IO).contains(newManifest));
   }
 
+  @Test
+  public void testRecoveryWithManualReCommit() {
+    TestTables.TestTableOperations ops = table.ops();
+    ops.failCommits(5);
+
+    AppendFiles append = table.newFastAppend().appendFile(FILE_B);
+    Snapshot pending = append.apply();
+    ManifestFile newManifest = pending.allManifests(FILE_IO).get(0);
+    Assert.assertTrue("Should create new manifest", new File(newManifest.path()).exists());
+
+    Assertions.assertThatThrownBy(append::commit)
+            .isInstanceOf(CommitFailedException.class)
+            .hasMessage("Injected failure");
```

Comment:
Looks like this is likely failing the style checks. You can run `./gradlew spotlessApply` to fix it.